tests/unit/plugins/modules/test_xfconf.yaml

```
@@ -21,11 +21,23 @@
     previous_value: '100'
     type: int
     value: '90'
+    version: "4.18.1"
   mocks:
     run_command:
-    - command: [/testbin/xfconf-query, --channel, xfwm4, --property, /general/inactive_opacity]
+    - command: [/testbin/xfconf-query, --version]
       environ: &env-def {environ_update: {LANGUAGE: C, LC_ALL: C}, check_rc: false}
       rc: 0
+      out: &version-output |
+        xfconf-query 4.18.1
+
+        Copyright (c) 2008-2023
+        	The Xfce development team. All rights reserved.
```

It might be better to write the tab as `\t`, if that's possible in `|` strings (I'm not sure about that). Otherwise this looks like a whitespace error, since this line mixes tabs and spaces before first non-whitespace character.

---

I have copied straight from the terminal, but this line is completely irrelevant to the module, so I am changing to spaces.
```suggestion
                The Xfce development team. All rights reserved.
```